router/logs.go

```diff
@@ -139,7 +139,7 @@ func createMessage(log model.Log, bodyCount int, item model.Item, user model.Use
 	itemInfo := fmt.Sprintf("[%v](https://%v/items/%v)", item.Name, os.Getenv("HOST"), item.ID)
 	if item.Type != 0 {
 		purpose := ""
-		count := math.Abs(float64(bodyCount))
+		count := int64(math.Abs(float64(bodyCount)))
```

Comment:
```suggestion
		count := int(math.Abs(float64(bodyCount)))
```
`bodyCount`は`int`なのでfloat->absの処理の後も`int`で表現できるのでわざわざ`int64`を指定しなくていいです